object_detection/detectron2_training-kfold.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
where appropriate also incorporate my suggested changes for the other notebook into this one